docs/general/create-account.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Create account
+sidebar_position: 1
+sidebar_label: Create account
+---
+
+The Vara Network is substrate-based, which means that any keypair compatible with the standards of the `subkey` module will also work on Vara.
```

Comment:
in fact address spaces depends only on crypto curves and for our case its common for substrate sr25519